tests/Core.UnitTests/Commands/ScheduledCommandsTest.cs

```diff
@@ -9,52 +10,96 @@ public class ScheduledCommandsTest: TestBase
     public ScheduledCommandsTest(ITestOutputHelper @out) : base(@out)
     { }
 
-    [Fact(Skip = "Flaky")]
+    [Fact]
     public async Task EnqueueEventOnCommandCompletion()
     {
         await using var services = new ServiceCollection()
+            .AddLogging()
             .AddFusion()
-            .AddLocalCommandScheduler()
+            .AddLocalCommandScheduler(Queues.Default)
             .AddComputeService<ScheduledCommandTestService>()
             .Services
             .BuildServiceProvider();
         await services.HostedServices().Start();
 
+        var queue = services.GetRequiredService<ICommandQueues>().Get(Queues.Default) as LocalCommandQueue;
         var testService = services.GetRequiredService<ScheduledCommandTestService>();
         var commander = services.GetRequiredService<ICommander>();
 
         testService.ProcessedEvents.Count.Should().Be(0);
         await commander.Call(new TestCommand(null));
         testService.ProcessedEvents.Count.Should().Be(0);
 
-        await Task.Delay(500);
+        await Awaiter.WaitFor(() => queue!.CompletedCommandCount != 0);
+
         testService.ProcessedEvents.Count.Should().Be(1);
     }
 
-    [Fact(Skip = "Flaky")]
+    [Fact]
     public async Task MultipleEventHandlersAreCalled()
     {
         await using var services = new ServiceCollection()
+            .AddLogging()
             .AddCommander()
-            .AddLocalEventHandlers()
             .AddHandlers<DedicatedInterfaceEventHandler>()
             .Services
             .AddSingleton<DedicatedInterfaceEventHandler>()
             .AddFusion()
-            .AddLocalCommandScheduler()
+            .AddLocalCommandScheduler(Queues.Default)
             .AddComputeService<ScheduledCommandTestService>()
             .AddComputeService<DedicatedEventHandler>()
             .Services
             .BuildServiceProvider();
         await services.HostedServices().Start();
 
+        var queue = services.GetRequiredService<ICommandQueues>().Get(Queues.Default) as LocalCommandQueue;
         var testService = services.GetRequiredService<ScheduledCommandTestService>();
         var commander = services.GetRequiredService<ICommander>();
 
         testService.ProcessedEvents.Count.Should().Be(0);
         await commander.Call(new TestCommand2());
 
-        await Awaiter.WaitFor(() => testService.ProcessedEvents.Count == 2);
-        testService.ProcessedEvents.Count.Should().Be(2);
+        await Awaiter.WaitFor(() => queue!.CompletedCommandCount == 2);
+        // await Awaiter.WaitFor(() =>  testService.ProcessedEvents.Count == 3);
+
+        await Task.Delay(500);
```

Comment:
What is this 500ms delay needed? `Awaiter.WaitFor` does not solve race condition?